cmd/cosign/cli/importkeypair/import_key_pair.go

```diff
@@ -36,7 +36,12 @@ func ImportKeyPairCmd(ctx context.Context, keyVal string, args []string) error {
 		return err
 	}
 
-	if cosign.FileExists("import-cosign.key") {
+	fileExists, err := cosign.FileExists("import-cosign.key\000x")
```

Comment:
the `\000x` is expected?

Reply:
that was a mistake. removed in [9d30c9e](https://github.com/sigstore/cosign/pull/2162/commits/9d30c9e081c91102c135b15f8d0c154873a3031b)